server/models/service-instance.js

```diff
@@ -140,9 +141,9 @@ module.exports = function extendServiceInstance(ServiceInstance) {
       if (!instance)
         return callback(Error(fmt('Instance with ID %s not found', instanceId
           )));
-
-      instance.applicationName = instInfo.appName;
-      return instance.save(callback);
+      return instance.updateAttributes(
+        {applicationName: instInfo.appName}, callback
```

Comment:
Now that I'm not on my phone.. this is the line split that looked odd to me. But it's just whitespace, so I can always fix it myself later if I really want to :-)